docs/source/secured_private_asset_transfer_tutorial.md

```diff
@@ -0,0 +1,394 @@
+# Secured asset transfer in Fabric
+
+This tutorial will demonstrate how an asset can be represented and traded between organizations in a Hyperledger Fabric blockchain channel, while keeping details of the asset and transaction private using private data, implicit date collections state based endorsement, transient data, and key based ACLs.
```

Comment:
"implicit date" is a typo.

Note - the way I originally structured this, I was keeping this first sentence simple and then putting this list of features used at the end of the next paragraph.